### PR TITLE
Scaffolding Rails

### PR DESCRIPTION
**Pull request by holgertidemand**
#### Description
Changes proposed in this pull request:

* adds scaffolded rails app

#### What I have learned working on this feature: 
Learned that we can pass in options when we create a rails app. 
```
rails new evensteven --database=postgresql --skip-test --skip-bundle
```

### CODE REVIEW

**Comment by davidboll** [comment]
You have to remove the comments from Gemfile buddy!

**Review by davidboll — APPROVED** [review]
OMG you're the best :)

**Review by tochman — CHANGES_REQUESTED** [review]
Needs to be updated with a PR description and link to PT story/chore

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+end
+
+group :development do
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: Please remove this

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+gem 'uglifier', '>= 1.3.0'
+
+gem 'coffee-rails', '~> 4.2'
+
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
+
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```
Comment (tochman) [inline Gemfile]: Please review what debugger you use. Pry is more in line with the standard we use at Craft

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+gem 'pg', '>= 0.18', '< 2.0'
+
+gem 'puma', '~> 3.7'
+
+gem 'sass-rails', '~> 5.0'
+
+gem 'uglifier', '>= 1.3.0'
+
+gem 'coffee-rails', '~> 4.2'
+
```
Comment (tochman) [inline Gemfile]: Please remove empty lines...

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
+production:
+  <<: *default
+  database: evensteven_production
+  username: evensteven
+  password: <%= ENV['EVENSTEVEN_DATABASE_PASSWORD'] %>
```
Comment (tochman) [inline config/database.yml:18]: Please clear comments from this file

**Review by tochman — APPROVED** [review]

**Review by tochman — COMMENTED** [review]
I still see the .idea files

**Pull request merged**